lib/PuppeteerSharp/DialogType.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PuppeteerSharp
+{
+    public enum DialogType
```

Comment:
we should not have this and the class `Dialog.Type`. This should replace the constants

Reply:
Agreed, How about now?